docs/language/ztests/language-operators-1.yaml

```
@@ -6,10 +6,6 @@
 # See https://github.com/brimdata/zed/pull/2901 for details
 # ============================================================================
 
-# The Zed being tested below has been modified from the way it appears in
-# docs/language/README.md, as "put" is not yet optional
-# (https://github.com/brimdata/zed/issues/2850).
-
 script: |
   zc -C 'widget | count() by color | put COLOR := to_upper(color)'
```

If you're going to delete the comment about #2850, then you need to delete the `put` here!
```suggestion
  zc -C 'widget | count() by color | COLOR := to_upper(color)'
```